src/main/java/beans/EnvriomentVariableBean.java

```diff
@@ -0,0 +1,61 @@
+package beans;
+
+import javax.ejb.Singleton;
+
+@Singleton(name = "EnvriomentVariableEJB")
+public class EnvriomentVariableBean {
+
+    // Fitbit Info
+    private String fitbitClientId;
+    private String fitbitClientSecret;
+    private String fitbitClientCallback;
+
+    //Aberfitness Info
+    private String aberfitnessClientId;
```

Comment:
These should all be:
`private final static string ABER_FITNESS_CLIENT_ID = System.getenv("fitbitClientId");`

As they do not change in the application runtime

Reply:
Resolved in most recent commit